contracts/priv/unity.yml

```diff
@@ -1441,6 +1460,42 @@ components:
       required:
         - id
         - name
+    Cluster:
+      properties:
+        isAvailable:
```

Comment:
Do we need separate `isRegionAvailable` and `isProviderAvailable` fields? I don't expect that we'll be adding a new provider any time soon, but the UI has to do different things for unavailable providers than it does for unavailable regions.

I suppose it could infer that, if all regions for a provider are unavailable, then so is the provider, but is that what we want?

Reply:
Hmmm, I think you're probably right, but is that something we even want to worry about right now? We could, but we'd probably want to update vault to have both fields too.

Reply:
It already does, as does our `Cluster` struct.

Reply:
And, if our existing UI uses the `isProviderAvailable` flag, then I'd like to worry about it right now to preserve existing behavior.

Reply:
Well that does makes sense if vault already has it :P I'll update it, thanks!